domain/service_broker.go

```diff
@@ -199,7 +199,7 @@ type Binding struct {
 	BackupAgentURL  string          `json:"backup_agent_url,omitempty"`
 	VolumeMounts    []VolumeMount   `json:"volume_mounts"`
 	Endpoints       []Endpoint      `json:"endpoints,omitempty"`
-	Metadata        BindingMetadata `json:"metadata,omitempty"`
+	Metadata        BindingMetadata `json:"metadata"`
```

Comment:
question: I am not sure about these omitempty being removed. It would change behaviour for nil values.

Reply:
Thanks for the info. I hadn't realised that was the scenario. I did see the omitzero, but I guess that is not what we were doing either, so happy to keep the automated changes.